test/e2e/cypress/plugins/index.js

```diff
@@ -16,7 +16,26 @@
  * @type {Cypress.PluginConfig}
  */
 // eslint-disable-next-line no-unused-vars
+
+const http = require('http');
+
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  on('task', {
+    startAgentHeartbeat(agents) {
+      const {collector_host, collector_port} = config.env
+      agents.forEach((agentId) => {
+        setInterval(() => {
+          http.request({
+            host: collector_host,
+            path: `/api/hosts/${agentId}/heartbeat`,
+            port: collector_port,
+            method: 'POST'
+          }).end()
+        }, 5000)
```

Comment:
Maybe we could re-use the `HeartbetInterval` constant by passing it as an env var to cypress? in case we want to change that in the future, wdyt?
On the other hand, wouldn't it be safer to have it a bit under 5000ms in case of delays in the POST? haven't had a flaky test due to this but I guess it could happen?

Reply:
I added a `"heartbeat_interval": 5000` in our `cypress.json`, which then we use when starting heartbeating.

I would keep it `5000` as actually reflects what the agent does right now, and it's quite a lot of time to post a single event :sweat_smile: 

Then even if for some reason it takes more than 5s, it would still be good because trento already gives twice the time to actually make an hearbeat, so we have 10s of time to do that. A lot :trollface: 

`const HeartbeatTreshold = internal.HeartbeatInterval * 2`

Reply:
ok, I was actually proposing to couple it with the constant we have in go, but actually, what you did of making it a cypress setting makes more sense so that we can adjust it when needed (even to force fails). Nice!